backend/src/packages/chaiNNer_standard/image/io/save_image.py

```diff
@@ -35,6 +35,38 @@
 
 from .. import io_group
 
+
+class ImageFormat(Enum):
+    PNG = "png"
+    JPG = "jpg"
+    GIF = "gif"
+    BMP = "bmp"
+    TIFF = "tiff"
+    WEBP = "webp"
+    WEBP_LOSSLESS = "webp-lossless"
```

Comment:
Could we get rid of webp-lossless and instead just add a lossless checkbox that hides the quality slider?

Reply:
We can, but not in this PR.